tasks/main.yml

```diff
@@ -8,3 +9,36 @@
   tags:
     - vars
 
+- name: Install the VDO packages
+  ansible.builtin.dnf:
+    name: "{{ vdo_pkgs_list }}"
+    state: installed
+
+- name: Create the volume group
+  community.general.lvg:
+    vg: "{{ vdo_vg_name }}"
+    pvs: "{{ vdo_pv_disks }}"
+
+- name: Setup VDO storage pool
+  community.general.lvol:
+    vg: "{{ vdo_vg_name }}"
+    lv: "{{ vdo_lv_name }}"
+    size: "{{ vdo_phys_size }}"
+    opts: "--type vdo --vdosettings 'vdo_slab_size_mb={{ vdo_slab_size }}' --virtualsize {{ vdo_virt_size }}"
+
+- name: Create vdo mount point
+  ansible.builtin.file:
+    path: "{{ vdo_mount_point }}"
+    state: directory
+    mode: '0755'
+- name: Format the xfs filesystem
+  community.general.filesystem:
+    fstype: xfs
```

Comment:
That should be a variable as by default we use ext4, so at least make that one a variable too ?

Reply:
Added